### PR TITLE
Add drizzle

### PR DESCRIPTION
**Pull request by thibaultleouay**

### CODE REVIEW

File: apps/web/src/app/action.ts [hunk 1]
```diff
@@ -2,10 +2,11 @@
 
 import { Resend } from "resend";
 import { Redis } from "@upstash/redis";
+import { env } from "@/env.mjs";
 
 const redis = Redis.fromEnv();
 
-const resend = new Resend(process.env.RESEND_API_KEY);
+const resend = new Resend(env.RESEND_API_KEY);
```
Comment (mxkaske) [inline apps/web/src/app/action.ts:9]: Nice catch, need to get used to the t3 way!

File: packages/db/drizzle.config.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import type { Config } from "drizzle-kit";
+
+export default {
+  schema: "./src/schema/index.ts",
+  out: "./drizzle",
+  connectionString: process.env.DATABASE_URL || "",
+} satisfies Config;
```
Comment (mxkaske) [inline packages/db/drizzle.config.ts:8]: I like! 🔥

File: packages/db/src/db.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// db.ts
+import { drizzle } from "drizzle-orm/planetscale-serverless";
+
+import { connect } from "@planetscale/database";
+
+const config = {
+  host: process.env.DB_HOST,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+};
```
Comment (mxkaske) [inline packages/db/src/db.ts:10]: Why not using t3? We are missing them in the `.env.example` btw. 👀

Comment (thibaultleouay) [reply packages/db/src/db.ts:10]: The T3 env only seems to work on the next directory I should also look at the way to do it in a monorepo

**Pull request merged**